test/sol6/reserves/kyberFprReserveV2.js

```diff
@@ -0,0 +1,2991 @@
+const ConversionRates = artifacts.require("MockEnhancedStepFunctions.sol");
+const TestToken = artifacts.require("TestToken.sol");
+const WethToken = artifacts.require("WethToken");
+const Reserve = artifacts.require("KyberFprReserveV2");
+const MockSanityRates = artifacts.require("MockSanityRates");
+const NoPayableFallback = artifacts.require("NoPayableFallback");
+
+const Helper = require("../../helper.js");
+const reserveSetup = require("../../reserveSetup.js");
+const BN = web3.utils.BN;
+
+//global variables
+//////////////////
+const {precisionUnits, ethDecimals, ethAddress, zeroAddress, zeroBN, MAX_ALLOWANCE} = require("../../helper.js");
+const {expectEvent, expectRevert} = require('@openzeppelin/test-helpers');
+
+//balances
+let expectedReserveBalanceWei = new BN(0);
```

Comment:
should we have these as global parameters?
thinking we should aim for each test to be able to run separate.
in this case no sense of following reserve balance between tests.

Reply:
all tests can run separately, these params are global so we don't have to define for each test suite or pass them in validate function

Reply:
Ahh
I see